src/crons/status.checker/status.checker.service.ts

```diff
@@ -40,147 +49,193 @@ export class StatusCheckerService {
     private readonly mexTokenService: MexTokenService,
     private readonly mexEconomicService: MexEconomicsService,
     private readonly economicService: NetworkService,
-  ) { }
+    private readonly apiConfigService: ApiConfigService
+  ) {
+    this.initializeThresholds();
+  }
+
+  private initializeThresholds() {
+    switch (this.apiConfigService.getNetwork()) {
+      case 'mainnet':
```

Comment:
please move these settings in the config files, as an object with attributes 🙏

Reply:
done